static/app/views/releases/detail/overview/index.tsx

```diff
@@ -324,6 +324,8 @@ class ReleaseOverview extends AsyncView<Props> {
                   version={version}
                   location={location}
                   defaultStatsPeriod={defaultStatsPeriod}
+                  queryFilterDescription={t('In this release')}
```

Comment:
do we need to do prop drilling here or could we just add `queryFilterDescription={t('In this release')}` to the `GroupList` in `overview/issues.tsx`?

Reply:
No you are right.